Unable to build Docker Image from master branch

### Describe the bug

I wanted to customize the Metabase OSS Edition and build the docker image

But I got an error while building frontend

### To Reproduce

1. docker build .

### Expected behavior

Build normally

### Logs

        [:app] Compiling ...
        2025-10-23 13:08:19,175 INFO metabase.util :: Maximum memory available to JVM: 2.9 GB 
        [:app] Build completed. (293 files, 214 compiled, 0 warnings, 78.60s)
        $ WEBPACK_BUNDLE=production rspack build
        WARNING in ? Module Warning (from /home/node/node_modules/postcss-loader/dist/cjs.js):
          ? from "postcss-is-pseudo-class" plugin: Complex selectors in '[data-mantine-color-scheme='dark'] :is(.root:not(:has(.pickerSearch:focus)) [aria-rowindex="0"] [aria-colindex="0"])' can not be transformed to an equivalent selector without ':is()'.
          ? 
          ? Code:
          ?   [data-mantine-color-scheme='dark'] :is(.root:not(:has(.pickerSearch:focus)) [aria-rowindex="0"] [aria-colindex="0"]){
          ?       background:rgba(38, 38, 38, 0.8);
          ? }
          ?
        
        WARNING in ? Module Warning (from /home/node/node_modules/postcss-loader/dist/cjs.js):
          ? from "postcss-is-pseudo-class" plugin: Complex selectors in '[data-mantine-color-scheme='dark'] :is(.root:focus-within [data-active])' can not be transformed to an equivalent selector without ':is()'.
          ? 
          ? Code:
          ?   [data-mantine-color-scheme='dark'] :is(.root:focus-within [data-active]){
          ?       background:rgba(38, 38, 38, 0.8);
          ? }
          ?
        
        WARNING in ? Module Warning (from /home/node/node_modules/postcss-loader/dist/cjs.js):
          ? from "postcss-is-pseudo-class" plugin: Complex selectors in '[data-mantine-color-scheme='dark'] :is(.root:not(:has(.pickerSearch:focus)) [aria-rowindex="0"] [aria-colindex="0"])' can not be transformed to an equivalent selector without ':is()'.
          ? 
          ? Code:
          ?   [data-mantine-color-scheme='dark'] :is(.root:not(:has(.pickerSearch:focus)) [aria-rowindex="0"] [aria-colindex="0"]){
          ?       background:rgba(38, 38, 38, 0.8);
          ? }
          ?
        
        WARNING in ? Module Warning (from /home/node/node_modules/postcss-loader/dist/cjs.js):
          ? from "postcss-is-pseudo-class" plugin: Complex selectors in '[data-mantine-color-scheme='dark'] :is(.root:focus-within [data-active])' can not be transformed to an equivalent selector without ':is()'.
          ? 
          ? Code:
          ?   [data-mantine-color-scheme='dark'] :is(.root:focus-within [data-active]){
          ?       background:rgba(38, 38, 38, 0.8);
          ? }
          ?
        
        ERROR in ./admin/databases/components/DatabaseHelpSidePanel/EmbeddedEngineDocContent/useEngineDocMarkdownContent.tsx
          ? Cannot find module 'docs/databases/connections' for matched aliased key 'docs'
        
        1 ERROR in child compilations (Use 'stats.children: true' resp. '--stats-children' for more details)
        Rspack compiled with 2 errors and 4 warnings in 88.31 s
        
        iframe_sdk_embed_v1:
          iframe_sdk_embed_v1 compiled successfully in 2.24 s
        
        embedding_sdk_bundle:
          WARNING in ? asset size limit: The following asset(s) exceed the recommended size limit (244.141 KiB). This can impact web performance.
            ? Assets:
            ?   embedding-sdk.js (10.772 MiB)
        
          WARNING in ? entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244.141 KiB). This can impact web performance.
            ? Entrypoints:
            ?   main (10.772 MiB)
            ?       embedding-sdk.js
        
          WARNING in ? Rspack performance recommendations:
            ? You can limit the size of your bundles by using import() to lazy load some parts of your application.
            ? For more info visit https://rspack.rs/guide/optimization/code-splitting
        
          embedding_sdk_bundle compiled with 3 warnings in 86.87 s
        info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
        error Command failed with exit code 1.
        error Command failed with exit code 1.
        info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Step failed: Running build steps for Enterprise Edition version v0.56.11.1: version, translations, frontend, licenses, drivers, uberjar

### Information about your Metabase installation

```JSON
Not built
```

### Severity

critical

### Additional context

_No response_

--- a/.dockerignore
+++ b/.dockerignore
@@ -1,5 +1,4 @@
 .babel_cache
-docs/*
 target/*
 .circleci
 .cpcache

